h/templates/client/annotation.html

```diff
@@ -110,7 +110,7 @@
               class="dropdown-menu-btn"><i class="h-icon-check btn-icon"></i> Delete</button>
       <publish-annotation-btn
         class="publish-annotation-btn"
-        group="vm.group()"
+        group="::vm.group()"
```

Comment:
I think this one breaks the UI when creating a new annotation and changing between groups without saving it